src/query/plan/operator.cpp

```
@@ -321,6 +327,12 @@ bool CreateExpand::CreateExpandCursor::Pull(Frame &frame, ExecutionContext &cont
 
   if (!input_cursor_->Pull(frame, context)) return false;
 
+  if (context.auth_checker && (!context.auth_checker->Accept(*context.db_accessor, self_.edge_info_.edge_type,
+                                                             memgraph::auth::FineGrainedPermission::CREATE_DELETE) ||
+                               !context.auth_checker->Accept(*context.db_accessor, self_.node_info_.labels,
```

Check with Boris if we need to rather throw an exception here.